src/Redux/Books/books.js

```diff
@@ -0,0 +1,40 @@
+export const ADD_BOOK = 'Books/books/ADD_BOOK';
+export const REMOVE_BOOK = 'Books/books/REMOVE_BOOK  ';
+
+export const initialState = [
+  {
+    id: 1,
+    title: 'hello world',
+    author: 'Hello',
+  },
+  {
+    id: 2,
+    title: 'hello Books',
+    author: 'book',
+  },
+];
+
+const Reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.books];
+
+    case REMOVE_BOOK:
+      return state.filter((books) => action.booksId !== books);
+
+    default:
+      return state;
+  }
+};
+
+export const AddBook = (books) => ({
+  type: 'ADD_BOOK',
+  books,
+});
+
+export const RemoveBook = (booksId) => ({
+  type: 'REMOVE_BOOK',
+  booksId,
+});
+
+export default Reducer;
```

Comment:
- [ ] Kindly make sure your application is reading correctly as right now I can't test it because of the error in the image below. it shows a blank page and the error in the console. Make sure you fix it:
![Screenshot from 2022-09-27 21-14-27](https://user-images.githubusercontent.com/83042742/192616176-329b2610-10f7-4825-9a50-e922ae391435.png)